index.d.ts

```diff
@@ -0,0 +1,24 @@
+import {type Buffer} from 'node:buffer';
+
+/**
+Strip the final [newline character](https://en.wikipedia.org/wiki/Newline) from a string/Buffer/Uint8Array.
+
+@param input
```

Comment:
This has no purpose without a description.

Reply:
Fixed, by removing this line.